src/sensai/featuregen.py

```diff
@@ -542,3 +599,36 @@ def _generate(self, df: pd.DataFrame, ctx=None) -> pd.DataFrame:
         else:
             _log.info(f"Generating target features via {self.vectorModel.__class__.__name__}")
             return self.vectorModel.predict(df)
+
+
+def flattenFeatureGenerator(fgen: FeatureGenerator, columnsToFlatten: List[str] = None,
```

Comment:
I would propose the name "flattenedFeatureGenerator", as it makes clear that it returns a new one and does not (somehow) flatten the existing one.
Perhaps even "FlattenedFeatureGenerator", as it's akin to a construction of a new instance (though some might disagree with the notion of naming a function this way in general).

Reply:
Done. Aaand, I'm merging this :)